src/main/java/org/kohsuke/github/GHIssue.java

```diff
@@ -158,10 +158,8 @@ public GHIssueState getState() {
      * Gets labels.
      *
      * @return the labels
-     * @throws IOException
-     *             the io exception
      */
-    public Collection<GHLabel> getLabels() throws IOException {
+    public Collection<GHLabel> getLabels() {
```

Comment:
@jglick @timja 
This is an API change, isn't it?

Reply:
Yes; binary compatible, potentially source-incompatible.